docs/scheduler-http-api.md

```diff
@@ -479,6 +479,32 @@ HTTP/1.1 202 Accepted
 
 ```
 
+### SUPPRESS
+Sent by the scheduler when it doesn't need offers for a given set of its roles. When Mesos master receives this request, it will stop sending offers for the given set of roles to the framework. As a special case, if roles are not specified, all subscribed roles of this framework are suppressed.
+
+Note that master continues to send offers to other subscribed roles of this framework that are not suppressed. Also, status updates about tasks, executors and agents are not affected by this call and tasks will continue running for `FrameworkInfo.failover_timeout`.
```

Comment:
I would remove `...and tasks will continue running for `FrameworkInfo.failover_timeout`` part. That timeout doesn't come into play at all during suppression. That will only come into play when a framework disconnects.